### DISCUSSION TITLE
Was gluonts.mx.model.deepar already there in the 0.10.2 version ?

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2421] [Q&A] [answered]
I work with the 0.10.2 version of GluonTs
Since I have some problems with the forecast, I wanted to try to import DeepAR differently.
So far I have used gluonts.model.deepar, now I wanted to try the following:
gluonts.mx.model.deepar 
but I can't import the latter at all. Is that because it doesn't work on the 0.10.2 version or what ?

**Comment by jaheba** [comment, marked as answer]
Hi, in `0.11` we moved mxnet based models from `gluonts.model` to `gluonts.mx.model`.

This means that you can import DeepAR in `0.10` using:

```py
from gluonts.model.deepar import DeepAREstimator
```

**Reply by MarleneBs** [reply]
perfect, thanks, so i continue as usual.
